src/displayapp/screens/settings/SettingHeartRate.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <cstdint>
+#include <lvgl/lvgl.h>
+
+#include "components/settings/Settings.h"
+#include "displayapp/screens/ScreenList.h"
+#include "displayapp/screens/Screen.h"
+#include "displayapp/screens/Symbols.h"
+#include "displayapp/screens/CheckboxList.h"
+
+namespace Pinetime {
+
+  namespace Applications {
+    namespace Screens {
+
+      struct Option {
+        const Pinetime::Controllers::Settings::HeartRateBackgroundMeasurementInterval interval;
+        const char* name;
+      };
+
+      class SettingHeartRate : public Screen {
+      public:
+        SettingHeartRate(Pinetime::Applications::DisplayApp* app, Pinetime::Controllers::Settings& settings);
```

Comment:
There is no need to take a `DisplayApp*` parameter anymore, nor store it in the class.

Reply:
fixed